pkgs/applications/networking/cluster/cert-manager-ctl/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "cert-manager-ctl";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "jetstack";
+    repo = "cert-manager";
+    rev = "v${version}";
+    sha256 = "0zi3wmjdp0zklyv1q3zcfpqhn0jb1xj03vl0mcpx6mjb7mqzvrmd";
+  };
+
+  vendorSha256 = "sha256-xLjYfnvrfAhtXsQT6+IabAcI+cZozABB3refAbX8jrE=";
+  runVend = true;
```

Comment:
https://github.com/NixOS/nixpkgs/blob/f557203a0468aad924fcfd6ef3216d6497125df5/pkgs/development/go-modules/generic/default.nix#L58

But it also seems to build without `proxyVendor`?

Reply:
Done.